app/controllers/cfp/events_controller.rb

```diff
@@ -100,23 +100,26 @@ def confirm
 
   def join
     @token = params[:token] || ''
+    @join_as = params[:join_as] || ''
```

Comment:
Wait a minute. Here you just copy the POST parameter join_as from the request … right?